bridge/polyfill/src/local-storage.ts

```diff
@@ -0,0 +1,8 @@
+/*
+* Copyright (C) 2022-present The WebF authors. All rights reserved.
+*/
+
+import {Storage, storageProxyHandler } from "./storage";
+
+// @ts-ignore
```

Comment:
Unnecessary comment. This isn't an error like `session-storage.ts`.

Reply:
done